conctr.agent.class.nut

```diff
@@ -10,8 +10,13 @@ class Conctr {
     static version = [1,0,0];
 
     static DATA_EVENT = "conctr_data";
+    static LOCATION_REQ = "conctr_get_location";
+    static LOCATION_RESP = "conctr_location";
+    static AGENT_OPTS = "conctr_agent_options";
+    static SOURCE_DEVICE = "impdevice";
```

Comment:
yeh, but this is the constant that shows the data came from the devices sendData function. On line 133 i check whether the source was not the device.